spec/requests/sprints_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+
+RSpec.describe 'Sprints API' do
+
+  let!(:course) { create(:course) }
+  let(:course_id) { course.id }
+  let!(:sprints) { create_list(:sprint, 20, course_id: course.id) }
+  let(:id) { sprints.first.id }
+
+  # Test suite for GET /courses/:course_id
+  describe 'GET /courses/:course_id with serialized sprints' do
+    before { get "/courses/#{course_id}" }
+
+    context 'when course exists' do
+      it 'returns status code 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'returns all course sprints' do
+        expect(json['sprints'].size).to eq(20)
+      end
+    end
+
+    context 'when course does not exist' do
```

Comment:
aren't we already testing this in the course API tests?

Reply:
The test here is checking the appearance of sprints inside a course.